.github/workflows/ci.yml

```diff
@@ -191,28 +191,16 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [14, 18]
-        postgres-version: [9.5, 10]
+        postgres-version: [oldest, latest]
```

Comment:
Should we do this for the other dialects as well? It's a bit odd having this only for postgres

Reply:
I started implementing these changes but I'd like to move it to its own PR that I can just discard if it doesn't work out